.circleci/retry.sh

```diff
@@ -1,23 +1,25 @@
 #!/bin/bash
 # Usage: retry.sh <max_tries> <command to run>
 
+tag="RETRY"
 readonly max_tries=$1
 readonly command="${@: 2}"
```

Comment:
I know this wasn't your code, but I think you need parentheses here. See [shellcheck's wiki](https://github.com/koalaman/shellcheck/wiki/SC2124). Might be a good thing to run on shell scripts we modify.

Reply:
It is my code actually, I'm just not an expert with bash scripts :).  Thanks for sharing the `shellcheck` tool - I just used it to verify the problem and confirm a fix!